### DISCUSSION TITLE
Vue.js ?

### DISCUSSION THREAD

**Discussion by Droppix** [wevm/wagmi#986] [Q&A]
Hi,

Is there a way to implement it in vue.js?

Thks

**Comment by jxom** [comment]
No first-class support for Vue just yet, but there is a community-led project at https://github.com/wobsoriano/vagmi which uses the wagmi core library (`@wagmi/core`)!

**Reply by IbrahimIjai** [reply]
Link isnt going through

**Comment by hellowodl** [comment]
https://github.com/unicape/use-wagmi
